circle.yml

```diff
@@ -20,15 +21,15 @@ machine:
     GUSH_USE_FS: true
 dependencies:
   cache_directories:
-    - "~/docker"
+    - "$HOME/docker"
   override:
     - docker build -t gush .
 test:
   override:
-    - composer install
-    - wget https://phar.phpunit.de/phpunit.phar && chmod +x phpunit.phar
-    - php phpunit.phar --verbose
-    #- docker run -it -e GUSH_CONFIG=`cat ./.gush.yml | base64` gush -vvvv --help
+    - composer install --prefer-dist --optimize-autoloader --no-interaction --no-scripts
```

Comment:
there is no need for --no-scripts because that is an ugly feature of composer and we don't use it